setup.py

```diff
@@ -71,6 +71,7 @@
         "filelock>=3.0,<3.1",
         "lmdb",
         "more-itertools",
+        "checklist",
```

Comment:
We should probably pin this
```suggestion
        "checklist==0.0.10",
```